test/test_tree_3d_mhd.jl

```diff
@@ -81,6 +81,18 @@ EXAMPLES_DIR = pkgdir(Trixi, "examples", "tree_3d_dgsem")
       l2   = [0.0186712969755079, 0.01620736832264799, 0.01620736832264803, 0.016207474382769683, 0.07306422729650594, 0.007355137041002365, 0.0073551370410023425, 0.00735520932001833, 0.000506140942330923],
       linf = [0.28040713666979633, 0.27212885844703694, 0.2721288584470349, 0.2837380205051839, 0.7915852408267114, 0.08770240288089526, 0.08770240288089792, 0.08773409387876674, 0.050221095224119834])
   end
+
```

Comment:
These test values will need updated once the corrected viscous fluxes are implemented.

Reply:
They have been updated.